lib/osf-components/addon/components/search-result-card/template.hbs

```diff
@@ -86,7 +86,7 @@
             </div>
         {{/if}}
     </div>
-    <CpPanel @open={{this.isOpenSecondaryMetadata}} as |panel|>
+    <CpPanel local-class='cp-panel' @open={{this.isOpenSecondaryMetadata}} as |panel|>
         <panel.body>
             <hr>
```

Comment:
If you're adding the border to the panel, do we still need the `<hr>` element?

Reply:
Possibly not, however it has not been showing on OSF. Removing it now and testing